wwwapp/forms.py

```diff
@@ -32,16 +33,44 @@ def __init__(self, *args, **kwargs):
         self.helper = FormHelper(self)
         self.helper.label_class = 'col-lg-3'
         self.helper.field_class = 'col-lg-9'
-    
+
+    def clean_start_date(self):
+        if self.cleaned_data['start_date']:
+            if self.cleaned_data['start_date'] < settings.WORKSHOPS_START_DATE:
+                raise ValidationError('Warsztaty rozpoczynają się ' + str(settings.WORKSHOPS_START_DATE))
+            if self.cleaned_data['start_date'] > settings.WORKSHOPS_END_DATE:
+                raise ValidationError('Warsztaty kończą się ' + str(settings.WORKSHOPS_END_DATE))
+            if 'end_date' in self.cleaned_data and self.cleaned_data['end_date']:
+                if self.cleaned_data['start_date'] > self.cleaned_data['end_date']:
+                    raise ValidationError('Nie możesz wyjechać wcześniej niż przyjechać! :D')
+        return self.cleaned_data['start_date']
+
+    def clean_end_date(self):
+        if self.cleaned_data['end_date']:
+            if self.cleaned_data['end_date'] < settings.WORKSHOPS_START_DATE:
+                raise ValidationError('Warsztaty rozpoczynają się ' + str(settings.WORKSHOPS_START_DATE))
+            if self.cleaned_data['end_date'] > settings.WORKSHOPS_END_DATE:
+                raise ValidationError('Warsztaty kończą się ' + str(settings.WORKSHOPS_END_DATE))
+            if 'start_date' in self.cleaned_data and self.cleaned_data['start_date']:
+                if self.cleaned_data['start_date'] > self.cleaned_data['end_date']:
+                    raise ValidationError('Nie możesz wyjechać wcześniej niż przyjechać! :D')
+        return self.cleaned_data['end_date']
+
     class Meta:
         model = UserInfo
-        fields = ['pesel', 'address', 'tshirt_size', 'comments']
+        fields = ['pesel', 'address', 'start_date', 'end_date', 'tshirt_size', 'comments']
         labels = {
             'pesel': 'Pesel',
             'address': 'Adres zameldowania',
+            'start_date': 'Data przyjazdu :-)',
+            'end_date': 'Data wyjazdu :-(',
```

Comment:
```suggestion
            'end_date': 'Data wyjazdu',
```
Remove this emoji pls